tasks/main.yml

```diff
@@ -50,6 +50,15 @@
   tags:
   - install
 
+- name: Install the required dependencies (MacOS)
+  homebrew:
+    name:
+      - openssl@1.1
+      - python3
+    state: present
+  become: no
+  when: ansible_os_family == 'Darwin'
+
```

Comment:
This is a query, I have been running this role for 18 months and want to disable this step with `--skip-tags "install". Brew is a really opinionated pkg manager and I really need to avoid using it. I query whether you need this at line 61.
```
tags:
  - install
```